apis/lightspeed/v1beta1/webhook_suite_test.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2025.
```

Comment:
we do all envtest via `test/functional` so could either drop this file or shorten it as https://github.com/openstack-k8s-operators/openstack-operator/blob/main/apis/core/v1beta1/webhook_suite_test.go ?

Reply:
Ok, thanks, I'll update the file!